### DISCUSSION TITLE
"plain text" or "markdown" top level note type (not "code" sub-type)?

### DISCUSSION THREAD

**Discussion by krc** [TriliumNext/Trilium#3843] [Ideas]
I think a top level "plain text", or possibly "markdown" note type to go along with "text".

Aside from the more semantic difference of not being under "code", since it is not meant to be code, there are some disadvantages to the code/plain text or code/markdown types

- the code notes don't get automatic TOC sidebar like text notes (based on md '#' headings)
- the code notes flash as you type words, the word is highlighted and dehighlights and rehighlights with each letter typed in the word

Some light markdown highlighting would be nice (like what already happens in the code/markdown note type), but a full rendering view isn't needed necessarily in my opinion. 

I have been looking at maybe migrating from Joplin, some of the extra abilities of Trilium are really nice, but I use the "code editor"/non-formatted markdown pane like 90% of the time, that's how I like my notes unless a specific note needs more advanced features.

Failing a top level plain text or markdown type, a way to import MD files to type code/markdown would be nice. And the flashing of the code editor really needs to be fixed in any case...

**Comment by krc** [comment]
I filed the flashing as a bug: https://github.com/zadam/trilium/issues/3844

**Comment by zadam** [comment]
I think the basic problem here boils down to you wanting to use Trilium as something which it isn't - a Markdown editor.

Having two top level "text" types is just confusing. If you want to write plain text without formatting, you can do it without issue in "Text" note. Just don't make the text bold etc. Yes, it's still stored as HTML, but that's irrelevant from the user point of view.
